apps/docs/pages/guides/platform/custom-domains.mdx

```diff
@@ -12,7 +12,7 @@ Custom Domains are currently in beta, and are being slowly made available to pro
 
 </Admonition>
 
-Custom domains allow you to present a branded experience to your users. You can also use a subdomain for the purposes of this guide.
+Custom domains allow you to present a branded experience to your users. Currently, you must use a subdomain (e.g. `api.example.com`, rather than `example.com`) for the purposes of this guide.
```

Comment:
```suggestion
Custom domains allow you to present a branded experience to your users. Currently, you must use a subdomain (e.g., `api.example.com`, rather than `example.com`) for the purposes of this guide.
```